test/IntegrationTests/TicketService/test_create_custom_fields.py

```diff
@@ -0,0 +1,160 @@
+            "field_name": "test_field",
+            "field_type": "VARCHAR(255)",
+            "required": True
+        }
+        logger.info(f"Testing invalid customer GUID with data: {data}")
+        response = requests.post(url, json=data)
+        self.assertEqual(response.status_code, 400, "Invalid customer_guid should result in 400")
+        self.assertIn("Database", response.text)
+        logger.info("Test case for invalid customer_guid passed.")
+
+    def test_add_duplicate_custom_field_name(self):
+        """Test adding a custom field with a duplicate field name."""
+        url = f"{self.BASE_URL}/custom_fields"
+        data = {
+            "customer_guid": self.valid_customer_guid,
+            "field_name": "duplicate_field",
+            "field_type": "VARCHAR(255)",
+            "required": True
+        }
+
+        # Add the custom field for the first time
+        logger.info(f"Adding initial custom field with data: {data}")
+        response = requests.post(url, json=data)
+        self.assertEqual(response.status_code, 200, "Failed to add the initial custom field")
+
+        # Try adding the same custom field again
+        logger.info(f"Testing duplicate custom field name with data: {data}")
+        response = requests.post(url, json=data)
+        self.assertEqual(response.status_code, 400, "Duplicate field name should result in 400")
+        self.assertIn(f"A custom field with this name {data['field_name']} already exists.", response.text)
+        logger.info("Test case for duplicate custom field name passed.")
+
+    def test_add_custom_field_invalid_field_type(self):
+        """Test adding a custom field with an invalid field type."""
+        url = f"{self.BASE_URL}/custom_fields"
+        invalid_field_type = "INVALID_TYPE"
+        data = {
+            "customer_guid": self.valid_customer_guid,
+            "field_name": "invalid_field_type_test",
+            "field_type": invalid_field_type,
+            "required": True
+        }
+
+        # Attempt to add a custom field with an invalid field type
+        logger.info(f"Testing invalid field type with data: {data}")
+        response = requests.post(url, json=data)
+
+        # Assert that the API returns a 400 status code
+        self.assertEqual(response.status_code, 400, "Invalid field type should result in 400")
+
+        # Assert that the error message contains the expected text
+        expected_message = f"Unsupported field type: {invalid_field_type}. Allowed types are:"
+        allowed_types = ["DATE", "INT", "VARCHAR(255)", "MEDIUMTEXT", "BOOLEAN"]
+
+        # Check if all allowed types are in the response text
+        for allowed_type in allowed_types:
+            self.assertIn(allowed_type, response.text, f"'{allowed_type}' not found in response")
+
+        logger.info("Test case for invalid field type passed.")
+
+    def test_database_server_down(self):
```

Comment:
Lets remove this testcase     def test_database_server_down(self):

Reply:
removed sir